VoxelSniperBukkit/build.gradle.kts

```diff
@@ -0,0 +1,25 @@
+plugins {
+    id("voxel-common")
```

Comment:
It seems like you're mixing core and common?

Reply:
I might restructure it a bit, but the idea is that we have a common gradle file for all 'modules', so that we don't need to change the version of a dependency in multiple places. Might rename it to voxel-core, and might add a second one with some specific configuration for the implementations.

Reply:
https://github.com/KevinDaGame/VoxelSniper-Reimagined/pull/51/commits/f96d57bce9c9008b31c6de3254f098a0392572c5